Packs/CoreAlertFields/IncidentFields/incidentfield-Initiated_By.json

```diff
@@ -39,11 +39,10 @@
           "cliName": "carbonblackesprocessname",
           "type": "shortText"
         },
-       
+        
         {
-            "cliName": "digitalguardianprocessname",
-            "type": "shortText"
+            "cliName": "digitalguardianparentprocessname",
```

Comment:
Initiate by is the process name not parent process name
Should be aliased to CGO name (causality_actor_process_image_name)

Reply:
Sure - changed the x2_filed map + changed the alias to the  `digitalguardianprocessname`